### PR TITLE
final commit

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

**Review by ManSOorcode — APPROVED** [review]
keep going.

**Review by murtazshaikh — APPROVED** [review]
Approved

File: style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  text-align: center;
+  line-height: 50px;
+  color: #fff;
+
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  background: #007ce0;
+  pointer-events: none;
```
Comment (Abbas4tech) [inline style.css:45]: What's the use of `pointer-events` ?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  border: none;
+
+  box-shadow: 0 5px 25px rgba(0, 0, 0, 0.5);
+  appearance: none;
+  outline: none;
+}
+.box:before {
+  width: 50px;
+  height: 50px;
```
Comment (Abbas4tech) [inline style.css]: Use relative units

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Need some refactoring

File: style.css (hunk 1 above)
Comment (nehalskdev) [reply style.css:45]: The ```pointer-events``` property allows you to control whether an element can be the target of pointer (mouse or touch) events.

File: style.css (hunk 2 above)
Comment (nehalskdev) [reply style.css]: surely it is done please check @Abbas4tech

**Comment by MTahir001** [comment]
LGTM bro!!!

**Pull request merged**